cli/ballerina-cli/src/main/resources/cli-help/ballerina-init.help

```diff
@@ -32,4 +31,4 @@ DESCRIPTION
 
 EXAMPLES
      Initialise a package in the current directory.
```

Comment:
Other commands have more space for indentation of the explanation.

Reply:
Fixed